modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/info/item/provider/test/DDMFormValuesInfoFieldValuesProviderTest.java

```diff
@@ -105,6 +122,52 @@ public void testGetInfoFieldValuesCheckboxMultipleDDMFormFieldType()
 				).build()));
 	}
 
+	@Test
+	public void testGetInfoFieldValuesFieldSetDDMFormFieldType()
+		throws Exception {
+
+		DDMStructureTestHelper ddmStructureTestHelper =
+			new DDMStructureTestHelper(
+				PortalUtil.getClassNameId(DataDefinitionContentType.class),
+				_group);
+
+		DDMStructure ddmStructure = ddmStructureTestHelper.addStructure(
+			PortalUtil.getClassNameId(DataDefinitionContentType.class),
+			RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+			read("test-ddm-structure-definition-with-nested-fields.json"),
+			StorageType.DEFAULT.getValue());
+
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext(
+				_group.getGroupId(), TestPropsValues.getUserId());
+
+		JournalFolder journalFolder = _addJournalFolder(
+			ddmStructure.getStructureId(), serviceContext);
+
+		DDMTemplate ddmTemplate = DDMTemplateTestUtil.addTemplate(
+			_group.getGroupId(), ddmStructure.getStructureId(),
+			ClassNameLocalServiceUtil.getClassNameId(JournalArticle.class));
+
+		String content = read("ddm-structure-fieldset-field.xsd");
+
+		Map<Locale, String> nameMap = HashMapBuilder.put(
+			LocaleUtil.US, RandomTestUtil.randomString()
+		).build();
```

Comment:
You can replace this variable for `RandomTestUtil.randomLocaleStringMap()` and inline where you are using.

Reply:
Okay.